ctam/ctam.py

```diff
@@ -117,148 +119,179 @@ def get_exception_details(exec: Exception = ""):
 
 def main():
     args = parse_args()
-    # try:
-    # builds hierarchy of test groups and associated test cases
-    #ms_internal_tests
-
-    if args.version:
-        print(f"CTAM - version {__version__}")
-        exit()
-
-    if not args.workspace:
-        ifc_dir = os.path.join(os.path.dirname(__file__), "interfaces")
-        ext_test_root_dir =  os.path.join(os.path.dirname(__file__), "tests")
-        test_hierarchy = TestHierarchy(ext_test_root_dir, ifc_dir)
+    try:
+        # builds hierarchy of test groups and associated test cases
+        #ms_internal_tests
+
+        if args.version:
+            print(f"CTAM - version {__version__}")
+            exit()
+
+        default_config_path = os.path.join(os.path.dirname(os.path.dirname(__file__)), "json_spec", "input")
```

Comment:
@v-ajitpanda please add this line after 
`default_config_path = default_config_path.replace('/tmp/', '') if default_config_path.startswith('/tmp/') else default_config_path
     ` It causes error in our binary

Reply:
Added the above code